src/test/AddRemove.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment jsdom
+ */
+import Task from '../storage.js';
+
+document.body.innerHTML = `
+ <div id="container">
+
+<h1 id="head">Today's To Do</h1>    
+    <form>
+<input class="todo" type="text" placeholder="Add to your list..." autofocus />
+   <i id="add" class="fas fa-reply"></i> 
+</form>
+<div id="tasks"></div>
+
+    <div class="clear"><a href="#">Clear all completed</a></div>
+</div>
+`;
+describe('add and remove', () => {
+  window.localStorage = Storage.prototype;
+  test('Add task', () => {
+    const todoList = new Task();
+    const newTodo = {
+      description: 'task1',
+      completed: false,
+      index: 1,
+    };
+    const newTodo2 = {
+      description: 'task2',
+      completed: false,
+      index: 2,
+    };
+    todoList.addTask(newTodo);
+    expect(todoList.list).toHaveLength(1);
+    todoList.addTask(newTodo2);
+    expect(todoList.list).toHaveLength(2);
+    expect(todoList.list[1].description).toBe('task2');
+  });
+
+  test('remove task', () => {
+    const todoList = new Task();
+    const newTodo = {
+      description: 'task3',
+      completed: false,
+      index: 3,
+    };
+    todoList.addTask(newTodo);
+    todoList.RemoveTask(newTodo.id);
+    expect(todoList.list[1].description).toBe('task2');
+    expect(todoList.list).toHaveLength(2);
+  });
```

Comment:
- [ ] Great job so far :clap: However, one test is still not passing, as you can see in the figure below :point_down:, please solve the problem with the "remove task" test and do ensure that all tests pass before requesting the second review
 
![test](https://user-images.githubusercontent.com/57562869/152143751-114334a0-c02f-4e2c-8517-c1eb6f3b56e2.png)